vocabularies/gsq-dataset-themes.ttl

```diff
@@ -0,0 +1,100 @@
+@prefix : <http://linked.data.gov.au/def/gsq-dataset-themes/> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/gsq-dataset-themes> a owl:Ontology .
+
+:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:modified "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:source "Developed by the Geological Survey of Queensland." ;
+    skos:definition "A list of broad thematic descriptions of datasets created, received, and held by the Geological Survey of Queensland."@en ;
+    skos:prefLabel "GSQ Dataset Themes"@en .
+
+:administrative-data a skos:Concept ;
+    skos:definition "Ancillary information regarding the administration of Queensland Resources such as adminstrivite boundaries, minng permits, legislation, and information about organisations such as DNRME and GSQ."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Administrative data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:geoscience-data a skos:Concept ;
+    skos:definition "Primary descriptive data defining the geological attributes of a feature of interest."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Geoscience data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:geoscience-project-data a skos:Concept ;
+    skos:definition "Data and media describing project information such as activities performed, planning and approvals for the collection and analysis of primary geoscience data."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Geoscience project data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:other a skos:Concept ;
+    skos:definition "Any other metadata, data, or information that is not described by any of the other GSQ Dataset Themes."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Other"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:operational-data a skos:Concept ;
+    skos:definition "Data describing the operations of mining, drilling, processing, and sales operations e.g. borehole drilling, pipelines, processing facilities."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Operational data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:physical-samples a skos:Concept ;
+    skos:definition "Data and metadata pertaining to physical samples taken during geoscience data acquisition e.g. dril core, rock chips, rock samples from outcrop, soil, sediment, water, drilling fluids, oil, gas etc."@en ;

```

Comment:
drill core typo in definition - "dril core"

Reply:
fixed